### PR TITLE
[master] sync FCP DB when restart sdkserver

### PR DESCRIPTION
**Pull request by jackydalong**
- Before this PR, FCP DB will not by updated when restarting sdkserver

- With this PR, FCP DB will be synced in the following order
First, sync with the fcp_list of zvmsdk.conf
Second, sync with FCP status queried by smcli System_WWPN_Query to z/VM

Signed-off-by: Da Long Wang <shdlwang@cn.ibm.com>

### CODE REVIEW

File: zvmsdk/volumeop.py [hunk 1]
```diff
@@ -762,6 +771,136 @@ def get_physical_wwpn(self, fcp_no):
...
+        # Get a dict of all FCPs in FCP DB
+        fcp_dict_in_db = self.get_fcp_dict_in_db()
+
+        # Update DB column of comment based on the info found in ZVM
+        for fcp in fcp_dict_in_db:
+            comment_dict = dict()
+            if fcp in fcp_dict_in_zvm:
+                # Possible state returned by ZVM:
+                # 'active', 'free' or 'offline'
+                comment_dict['state'] = fcp_dict_in_zvm[fcp].get_dev_status()
```
Comment (dongyanyang) [inline zvmsdk/volumeop.py:898]: print warning log when found fcp in abnormal state, eg: offline.

**Review by dongyanyang — COMMENTED** [review]
Generally lgtm, suggest some log enhancements.

File: zvmsdk/volumeop.py [hunk 2]
```diff
@@ -762,6 +771,136 @@ def get_physical_wwpn(self, fcp_no):
...
+            else:
+                LOG.info("FCP {} not found in ZVM.".format(fcp))
```
[the remark this answers is not in the document]
Comment (jackydalong) [reply zvmsdk/volumeop.py:900]: We have that info in below log, hence I prefer not print it alone here.
```
LOG.info("fcp_dict_in_db: {}".format(fcp_dict_in_db))
```

**Review by dongyanyang — APPROVED** [review]
LGTM

**Comment by dongyanyang** [comment]
Waiting UT

**Comment by jackydalong** [comment]
UT & PEP8 PASS
```
________ summary _______________________________________________________________
  py36: commands succeeded
  pep8: commands succeeded
  congratulations :)
  ```

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**